assets/style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+main {
+  background-image: url("../img/header-shapes\ mobile.png");
```

Comment:
[OPTIONAL] 
#### Please note that this suggestion is optional and not required towards the project completion. 

- [ ] **You have applied the background-image property to the main element. Applying it to the headline section might have been an easier approach**

As you will add more content in the future, the main element will grow in size. It will result in the background image expanding vertically on the website. Like in the image below. 

![image](https://user-images.githubusercontent.com/59168430/120851277-392dc800-c596-11eb-988d-9664bc3da13d.png)

You can apply a background image to the headline section. It will be easier to manage as the content grows.

Reply:
thank you for the review, I will take it in consideration.